ntc_templates/templates/index

```diff
@@ -11,6 +11,7 @@
 #
 Template, Hostname, Platform, Command
 
+alcatel_aos_show_vlan_port.textfsm, .*, alcatel_aos, show vlan (port|members)
```

Comment:
Is it possible to have shorthand commands on the AOS command line?
ex: `sh vl po`

Reply:
Technically you can do `sh vl po` if you do tab after each command. However, I will add the possibility :)